app/src/main/res/layout/activity_user_item_level3.xml

```diff
@@ -30,6 +80,7 @@
                 android:id="@+id/preview_button"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:fontFamily="@font/raleway_light"
```

Comment:
I forgot to ask you to change 'finish' button font as well, if it's OK :)

Reply:
I change the font. When you see finish text. I only change the text of the button